### DISCUSSION TITLE
Unable to configure the repository base url for v0.13.1

### DISCUSSION THREAD

**Discussion by dummybot88** [bentoml/BentoML#2480] [Q&A]
Hi Team,

I'm  currently unable to understand how to configure repository base url, so that whenever the bento service save() is called, the bento should get saved to S3 storage from AWS linux2 Ec2 instance.

Initially, I have configured my application to use bentoml `v0.10.0`, where I was able to configure the repository url by CLI command
`bentoml config set yatai_service.repository_base_url=s3://testingbentoml/` and run docker container for yatai server
`docker run -d -p 3000:3000 -p 50051:50051 \
    bentoml/yatai-service`

and then all my application's bento will go and get stored in the S3 bucket.

But with the latest bentoml version i.e. `v0.13.1,` I'm unable to achieve the same. I have configured a yml file with the below details:
`yatai:
  repository:
     type: s3
     s3:
       url: s3://testingbentoml/`

and then export the config file as `export BENTOML_CONFIG=/home/ec2-user/default_bentoml.yml` and then ran the docker container for yatai server. But no luck. I end up getting the below error:

```
Traceback (most recent call last):
  File "main.py", line 22, in <module>
    saved_path = iris_classifier_service.save('s3://testingbentoml/')
  File "/usr/local/lib/python3.8/site-packages/bentoml/service/__init__.py", line 749, in save
    return yc.repository.upload(self, version, labels)
  File "/usr/local/lib/python3.8/site-packages/bentoml/yatai/client/bento_repository_api.py", line 197, in upload
    return self.upload_from_dir(tmpdir, labels)
  File "/usr/local/lib/python3.8/site-packages/bentoml/yatai/client/bento_repository_api.py", line 207, in upload_from_dir
    get_bento_response = self.yatai_service.GetBento(
  File "/usr/local/lib64/python3.8/site-packages/grpc/_interceptor.py", line 216, in __call__
    response, ignored_call = self._with_call(request,
  File "/usr/local/lib64/python3.8/site-packages/grpc/_interceptor.py", line 257, in _with_call
    return call.result(), call
  File "/usr/local/lib64/python3.8/site-packages/grpc/_channel.py", line 343, in result
    raise self
  File "/usr/local/lib64/python3.8/site-packages/grpc/_interceptor.py", line 241, in continuation
    response, call = self._thunk(new_method).with_call(
  File "/usr/local/lib64/python3.8/site-packages/grpc/_channel.py", line 957, in with_call
    return _end_unary_response_blocking(state, call, True, None)
  File "/usr/local/lib64/python3.8/site-packages/grpc/_channel.py", line 849, in _end_unary_response_blocking
    raise _InactiveRpcError(state)
grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
        status = StatusCode.UNAVAILABLE
        details = "DNS resolution failed for testingbentoml/: C-ares status is not ARES_SUCCESS qtype=A name=testingbentoml/ is_balancer=0: Domain name not found"
        debug_error_string = "{"created":"@1652999795.000185254","description":"DNS resolution failed for testingbentoml/: C-ares status is not ARES_SUCCESS qtype=A name=testingbentoml/ is_balancer=0: Domain name not found","file":"src/core/lib/transport/error_utils.cc","file_line":165,"grpc_status":14}"

```

Could you please help me out in setting the bentoml in AWS linux2 ec2-instance that can save the bundle to S3?

Here is my Ec2 configuration:
Amazon Linux2 Ec2 Instance
Python Version : 3.8.5
Bentoml Version: 0.13.1
Java : 11
aws version: aws-cli/1.18.147 Python/2.7.18 Linux/4.14.275-207.503.amzn2.x86_64 botocore/1.18.6

PS: Surprisingly, when I simply ran the bentoml installation and configured the yml file in my Mac, even without running the yatai server docker container, I was able to list all the bundles that are previous available in the configured s3 bucket by executing the command `bentoml list` and was able to save the bundle directly on to the S3 storage. I have added my Mac configuration for reference.

Mac Conf:
Python version : 3.8.3
Bentoml version: 0.13.1
Java: 11
aws version : aws-cli/2.0.49 Python/3.7.4 Darwin/21.2.0 exe/x86_64

**Comment by yubozhao** [comment]
hey @dummybot88
Did you get a chance to try the new version of BentoML? `pip install --pre bentoml` and the new Yatai?  The 0.13 Bento does not compatible with the latest Yatai version
